.github/workflows/pr-cypress.yml

```diff
@@ -40,70 +40,58 @@ jobs:
         containers: [1, 2, 3]
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
-      # Restore the previous yarn modules and Cypress binary archives.
-      # Any updated archives will be saved automatically after the entire
-      # workflow successfully finishes.
-      # We use exact restore key to avoid yarn module snowballing
-      # https://glebbahmutov.com/blog/do-not-let-npm-cache-snowball/
-      - name: Cache central yarn modules
-        uses: actions/cache@v2
-        with:
-          path: ~/.cache/yarn
-          key: ${{ runner.os }}-node-${{ github.ref }}-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-node-${{ github.ref }}-${{ hashFiles('**/yarn.lock') }}
-
-      # We use the exact restore key to avoid Cypress binary snowballing
-      # https://glebbahmutov.com/blog/do-not-let-cypress-cache-snowball/
-      - name: Cache Cypress binary
-        uses: actions/cache@v2
+      - name: Setup Node
+        uses: actions/setup-node@v3
         with:
-          path: ~/.cache/Cypress
-          key: cypress-${{ runner.os }}-cypress-${{ github.ref }}-${{ hashFiles('**/package.json') }}
-          restore-keys: |
-            cypress-${{ runner.os }}-cypress-${{ github.ref }}-${{ hashFiles('**/package.json') }}
+          node-version-file: '.nvmrc'
+          cache: 'yarn'
 
-      # Cache local node_modules to pass to testing jobs
-      - name: Cache local node_modules
+      - name: Cache Node Modules
         uses: actions/cache@v2
         with:
           path: node_modules
           key: ${{ runner.os }}-node-modules-${{ github.ref }}-${{ hashFiles('**/yarn.lock') }}
           restore-keys: |
             ${{ runner.os }}-node-modules-${{ github.ref }}-
 
-      - name: Cache build folder
-        uses: actions/cache@v2
+      - name: Download Build Folder
+        if: ${{ !env.ACT }}
```

Comment:
Same here, maybe a comment re: why this is conditional.

Reply:
👍🏻

Reply:
This I am actually removing as artifacts work with act.